Observer/Multishipping/CheckoutControllerSuccessAction.php

```diff
@@ -93,8 +93,12 @@ public function execute(
     protected function processHostedRedirect(MultishippingResultInterface $multishippingResult)
     {
         $orderIds = [];
+        $quoteId = null;
         foreach ($multishippingResult->getOrders() as $order) {
             $orderIds[] = $order->getIncrementId();
+            if(!$quoteId && $order->getQuoteId()) {
+                $quoteId = $order->getQuoteId();
```

Comment:
Is there guaranteed to only be a single, consistent `quoteId`?

Reply:
It should, this is simply picking the first one, i can revise this if the tests pass with just first order quoteId

Reply:
the quoteId is consistent, the quote addresses and quote items are just assigned differently. so no change is needed here (we need to iterate over the orders anyway)